src/TextField/TextField.spec.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { render, cleanup } from "@testing-library/react";
+import "@testing-library/jest-dom/extend-expect";
+import { TextField } from "./TextField";
+
+afterEach(cleanup);
+
+test("when passed `autoFocus` prop, should have focus after mounting", () => {
+  const { getByTestId } = render(
+    <TextField autoFocus inputAs={<input data-testid="input" />} />
```

Comment:
nbd: Maybe worth adding another `TextField` to make the test more complete?

Reply:
I don't follow, what do you mean?

Reply:
I'm going to merge this as-is because I need it ASAP; I'll add more tests after we talk about it :)